### DISCUSSION TITLE
Changing Logo

### DISCUSSION THREAD

**Discussion by TzTqB** [GitSquared/edex-ui#1147] [Questions]
I want to customize eDEX-UI for my personal computer and was wondering how can I change the boot logo to something personal like my name, is there a way to change the boot logo of edex-ui in Linux. I am not a programmer so I might need some very basic instructions 😅
Thank you

**Comment by GitSquared** [comment]
The "logo" is not a real logo image, but a css-styled text - see https://github.com/GitSquared/edex-ui/blob/master/src/_renderer.js , line 291.

You can change that text by editing this file and then follow the README instruction to run your custom code instead of the distributed app build.
